src/components/action-popover/action-popover.test.tsx

```diff
@@ -22,9 +22,8 @@ jest.mock("../../__internal__/utils/helpers/guid");
   () => "guid-12345"
 );
 
-beforeAll(() => {
+beforeEach(() => {
```

Comment:
comment: we can just call this beforeAll and reinitialise real timers afterAll etc